web/src/app/users/SendTestDialog.tsx

```diff
@@ -111,6 +106,18 @@ export default function SendTestDialog(
     }
   }
 
+  const loading = (): boolean => {
+    // if the sendTestStatus.fetching is true OR there's no details and no error message OR the status is pending, return true
+    if (
+      sendTestStatus.fetching ||
+      (!details && !errorMessage) ||
+      status === 'pending'
+    ) {
+      return true
+    }
```

Comment:
nit: maybe break this into a const?

```js
const isLoading = sendTestStatus.fetching || (!details && !errorMessage) ||  status === 'pending'
```

A function could be nice to break apart, but may not be necessary (also with the `const` it doesn't have to be evaluated later -- less indirection)

```js
if (sendTestStatus.fetching) return true
if (!details && !errorMessage) return true
if (status === 'pending') return true

return false
```

Reply:
Refactored to a const.